tests/testthat/test-examine-constraints.R

```diff
@@ -27,32 +27,45 @@ test_that("`dm_examine_constraints()` works", {
   )
 
   # case of some constraints, all met:
-  walk(
-    dm_for_disambiguate_src,
-    ~ expect_identical(
-      dm_examine_constraints(.),
-      tibble(
-        table = c("iris_1", "iris_2"),
-        kind = c("PK", "FK"),
-        columns = new_keys("key"),
-        ref_table = c(NA, "iris_1"),
-        is_key = TRUE,
-        problem = ""
-      ) %>%
-        new_dm_examine_constraints()
-    )
+  expect_identical(
+    dm_examine_constraints(dm_for_disambiguate),
+    tibble(
+      table = c("iris_1", "iris_2"),
+      kind = c("PK", "FK"),
+      columns = new_keys("key"),
+      ref_table = c(NA, "iris_1"),
+      is_key = TRUE,
+      problem = ""
+    ) %>%
+      new_dm_examine_constraints()
+  )
+
+  # FIXME: maybe always include this test on a DB, despite PR #313?
```

Comment:
Please only include tests for one source, unless the test specifically tests two sources (e.g. `copy_dm_to()`).